src/components/definition-list/definition-list.pw.tsx

```diff
@@ -0,0 +1,335 @@
+    });
+  });
+
+  test("should verify Definition List text is displayed as a single column", async ({
+    mount,
+    page,
+  }) => {
+    await mount(<DLComponent dtTextAlign="left" asSingleColumn />);
+
+    const dl = getDataElementByValue(page, "dl");
+    await expect(dl).not.toHaveCSS("display", "grid");
+
+    const dt = getDataElementByValue(page, "dt");
+    await expect(dt.first()).toHaveCSS("text-align", "left");
+
+    await expect(dt.first()).not.toHaveCSS("grid-column", "1 / auto");
+
+    await assertCssValueIsApproximately(dt.first(), "width", 1366);
+
+    const dd = getDataElementByValue(page, "dd");
+    await expect(dd.first()).toHaveCSS("text-align", "left");
+
+    await expect(dd.first()).toHaveCSS("margin-left", "0px");
+
+    await expect(dd.first()).not.toHaveCSS("grid-column", "2 / auto");
+
+    await assertCssValueIsApproximately(dd.first(), "width", 1366);
+  });
+
+  test("should check Definition List definition is displayed with a tick icon", async ({
+    mount,
+    page,
+  }) => {
+    await mount(
+      <Dl>
+        <Dt>Text</Dt>
+        <Dd data-element="dd">
+          <Box display="inline-flex" alignItems="center">
+            <Box mr={1}>Details example</Box>
+            <Icon type="tick" />
+          </Box>
+        </Dd>
+      </Dl>
+    );
+
+    const dd = getDataElementByValue(page, "dd").locator(
+      '[data-component="icon"]'
+    );
+
+    await expect(dd).toHaveAttribute("data-element", "tick");
+  });
+
+  test("should verify Definition List is displayed with children inside React fragment", async ({
+    mount,
+    page,
+  }) => {
+    await mount(<DLReactFragment />);
+
+    const dt = getDataElementByValue(page, "dt");
+    await expect(dt).toHaveText("Text inside React Fragment");
+
+    const dd = getDataElementByValue(page, "dd");
+    await expect(dd).toHaveText("Description inside React Fragment");
+  });
+
+  test("should render Definition List within a box combined with typography and hr components", async ({
+    mount,
+    page,
+  }) => {
+    await mount(<DLBoxComponent />);
+
+    const box = getDataElementByValue(page, "box");
+    await expect(box.getByText("Segment Header")).toBeVisible();
+
+    await expect(box.locator("hr")).toBeVisible();
+
+    await expect(box.locator("dl")).toBeVisible();
+  });
+
+  test("should render dt and dd children when wrapped in a Box", async ({
+    mount,
+    page,
+  }) => {
+    await mount(<UsingBoxToOverrideBackgroundColor />);
+
+    const box1 = getDataElementByValue(page, "box1");
+    await expect(box1.first()).toBeVisible();
+
+    const box2 = getDataElementByValue(page, "box2");
+    await expect(box2.first()).toBeVisible();
+  });
+
+  test.describe("Accessibility tests for Definition List component", () => {
+    (alignValue as DlProps["dtTextAlign"][]).forEach((align) => {
+      test(`should pass accessibility tests for Definition List when text is ${align} aligned`, async ({
```

Comment:
Not a blocking issue, but as Ed mentioned previously, we don't need to include the component in every test description:
`should pass accessibility tests when text is ${align} aligned` - this is sufficient, and applies to all accessibility tests.